lib/govuk_ab_testing/minitest_helpers.rb

```diff
@@ -14,9 +16,33 @@ def with_variant(args)
         "You probably forgot to use `configure_response`"
 
       unless args[:assert_meta_tag] == false
-        content = ab_test.meta_tag_name + ':' + requested_variant.variant_name
+        expected_content =
+          ab_test.meta_tag_name + ':' + requested_variant.variant_name
         message = "You probably forgot to add the `analytics_meta_tag` to the views"
-        assert_select "meta[name='govuk:ab-test'][content='#{content}']", 1, message
+        meta_tags = css_select("meta[name='govuk:ab-test']")
+
+        assert_equal(1, meta_tags.count, message)
```

Comment:
This message will be a little less helpful than the old message (ie the old one will say exactly what CSS selector it was looking for?), although given that it's got a custom message this is probably okay?